apps/web/src/components/HyloCookieConsent/HyloCookieConsent.js

```diff
@@ -34,28 +47,28 @@ export default function HyloCookieConsent () {
   }
 
   return (
-    <div className={cn('HyloCookieConsent fixed bottom-0 h-16 left-0 w-full flex items-center text-background bg-foreground/60 border-t border-border px-10 z-[1000]', { [classes.showCookieInfo]: showCookieInfo })}>
-      <span className='font-bold flex-1'>
+    <div className={cn('HyloCookieConsent fixed bottom-0 h-16 left-0 w-full flex justify-between items-center text-foreground bg-input/80 px-2 sm:px-4 z-[1000]', { block: showCookieInfo })}>
+      <span className='font-bold flex-1 text-xs sm:text-base'>
         {t('Hylo uses cookies to enhance the experience.')}
         <button className={cn(classes.viewDetails, 'ml-4')} onClick={toggleShowCookieInfo}>{t('View details')}</button>
       </span>
-      <Button variant='secondary' onClick={acceptCookies}>{t('I Understand')}</Button>
-      <div className={cn(classes.cookieInformation, { [classes.showCookieInfo]: showCookieInfo })}>
-        <div className={classes.content}>
-          <div className={classes.pad}>
-            <h3>{t('How do we use cookies?')}</h3>
-            <h4>{t('Hylo login & session')}</h4>
+      <button className='p-2 rounded-md bg-selected text-foreground text-xs sm:text-base' onClick={acceptCookies}>{t('I Understand')}</button>
```

Comment:
why remove the use of the shadcn button component? it would be really nice to standardize the button component and the variants we use so we can tweak styling all in one place. components/ui/button